dkobo/hub/context_processors.py

```diff
@@ -19,3 +19,14 @@ def welcome_message(request):
         return ctx
     else:
         return {}
+
+def external_service_tokens(request):
+    context = {}
+    if settings.TRACKJS_TOKEN:
```

Comment:
This will raise an `AttributeError` if `TRACKJS_TOKEN` is totally absent (as opposed to being empty, etc.). Is that okay?